common.tf

```diff
@@ -3,6 +3,7 @@ locals {
   ingest_raw_cache_bucket_name                         = "${local.environment}-dr2-ingest-raw-cache"
   sample_files_bucket_name                             = "${local.environment}-dr2-sample-files"
   ingest_state_bucket_name                             = "${local.environment}-dr2-ingest-state"
+  reporting_bucket_name                                = "${local.environment}-dr2-reporting"
```

Comment:
1. Should this variable name start with "ingest"
2. Should "ingest" appear in the name (after "dr2")?